.travis.yml

```diff
@@ -133,7 +133,6 @@ jobs:
       script:
       - make test-subcommand
       - make test-integration
-      - make image-build-custom-scorecard-tests
```

Comment:
How the scorecard images used in the tests are built now? 

c/c @joelanford

Reply:
`image-build-custom-scorecard-tests` is required by `test-e2e-go`. No need to explicitly call it out in the travis config.

Reply:
Hi @joelanford,

so, we need to add this call to  test-e2e-go in order to get it merged.